docker/apps/agreement_database/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM vickumar/python-nodejs
+
+RUN git clone https://github.cfpb.gov/CFPB/agreement_database.git /apps/agreement_database/
```

Comment:
Right, the Dockerfile for each app would live within that app's repo.

Regarding images, we actually do have an internal DTR to store images that need to be shared. That being said, for local development, the main docker-compose process could still build all sub-app images locally, so there's no need for a central repository. This is how the grasshopper setup works.